cinema_service/settings.py

```diff
@@ -12,28 +1,18 @@
+import os
 from datetime import timedelta
 from pathlib import Path
 
-# Build paths inside the project like this: BASE_DIR / 'subdir'.
 BASE_DIR = Path(__file__).resolve().parent.parent
 
+SECRET_KEY = os.environ.get("SECRET_KEY",
+                            "django-insecure-6vubhk2$++agnctay_4pxy_8cq)"
+                            "mosmn(*-#2b^v4cgsh-^!i3")
 
-# Quick-start development settings - unsuitable for production
-# See https://docs.djangoproject.com/en/4.0/howto/deployment/checklist/
+DEBUG = os.environ.get("DEBUG", "True") == "True"
```

Comment:
The `DEBUG` setting should be set to `False` in production environments to prevent the display of sensitive information. Ensure this is controlled via environment variables.